src/jit/codegenxarch.cpp

```diff
@@ -263,10 +263,12 @@ BasicBlock* CodeGen::genCallFinally(BasicBlock* block, BasicBlock* lblk)
     }
     else
     {
+#ifndef JIT32_GCENCODER
         // Because of the way the flowgraph is connected, the liveness info for this one instruction
```

Comment:
This will get it to compile, but is there a TODO issue here related to GC handling of this nop on x86?